frontend/src/components/ImagePicker.js

```diff
@@ -3,12 +3,13 @@ import ImageUpload from './ImageUpload';
 import _ from 'underscore';
 
 const REG_VALID_URL = /^(https?:\/\/)?([\da-z\.-]+)\.([a-z\.]{2,6})([\/\w \.-]*)*\/?$/;
-const REG_VALID_TWITTER_USERNAME = /^[a-zA-Z0-9_]{1,15}$/;
+const REG_VALID_TWITTER_USERNAME = /^@?([a-zA-Z0-9_]{1,15})$/;
```

Comment:
Given the time constraint, I'm going to merge this and come back and find a fix for it.